diff --git a/jsx.d.ts b/jsx.d.ts
index afc1039e46d5339dc7241575f15f72183b3a2f4b..7b545d1c57f335c1b846df2d083b8fd3243b7150 100644
--- a/jsx.d.ts
+++ b/jsx.d.ts
@@ -11,7 +11,7 @@ declare global {
   namespace JSX {
     export interface Element extends VNode {}
     export interface ElementClass {
-      $props: {}
+      // $props: {}
     }
     export interface ElementAttributesProperty {
       $props: {}